pages/common/scala-cli.md

```diff
@@ -0,0 +1,24 @@
+# scala-cli
+
+> Tool to interact with the Scala programming language.
+> More information: <https://scala-cli.virtuslab.org/docs/overview/>.
+
+- Start a REPL (interactive shell) using a specific Scala and JVM version:
+
+`scala-cli --scala {{3.1.0}} --jvm {{temurin:17}}`
+
+- Compile and run a Scala script:
+
+`scala-cli run {{path/to/script.scala}}`
+
+- Compile and test a Scala script:
+
+`scala-cli test {{path/to/script.scala}}`
+
+- Format a Scala script:
```

Comment:
Does it format the file in-place or does it output the formatted contents to stdout?